Source/Shared/arcana/macros.h

```diff
@@ -7,7 +7,7 @@
 #define STRINGIFY_MACRO_IMPL(s) #s
 
 #define WSTRINGIFY_MACRO(s) WSTRINGIFY_MACRO_IMPL(s)
-#define WSTRINGIFY_MACRO_IMPL(s) L#s
+#define WSTRINGIFY_MACRO_IMPL(s) L""#s
```

Comment:
Where is this macro used?

Reply:
Or are these only used by consumers of the lib?

Reply:
I took that guess, yes as it's not used by BN. I'd have removed the macro if the fix was more difficult to find....and wait for complains :)